libs/utils/src/lib/spreadsheet/excel-data.ts

```diff
@@ -0,0 +1,57 @@
+import { utils, AOA2SheetOpts }  from 'xlsx';
+
+interface MergeOptions {
+  merge: MergeCells[]
+}
```

Comment:
```suggestion
interface AddLineOptions {
  merge: MergeCells
}
```

Why using `merge: MergeCells[]` as array as you only provide 1 item ?

Reply:
You can do multiple merge in one row (futur proof), I thought like this